assets/scripts/fe/src/app/components/payroll/payroll-list/payroll-list.component.html

```diff
@@ -14,22 +14,24 @@ <h5 class="subtitle mb-0">All Pay slips</h5>
 
   <div class="nav-list-style-1">
     <p class="text-center text-muted" *ngIf="payrollservice.plist.length === 0">No Pay slips available.</p>
-    <a uiSref="payroll-detail" [uiParams]="{id:p.id}" uiSrefActive="active" *ngFor="let p of payrollservice.plist">
-      <div class="nav-list-item d-flex justify-content-between align-items-start p-3">
-        <div class="d-flex align-items-center">
-          <input type="checkbox" name="" class="mr-3 inputcheck">
-          <div class="">
-            <h6 class="section-title mb-1">{{ p.payroll_code }}</h6>
-            <small class="text-muted">{{ p.date_created | date: '(EEE) MMM d, y' }}</small>
+    <div *ngFor="let p of payrollservice.plist">
+      <a uiSref="payroll-detail" [uiParams]="{id:p.id}" uiSrefActive="active" >
+        <div class="nav-list-item d-flex justify-content-between align-items-start p-3">
+          <div class="d-flex align-items-center">
+            <input *ngIf="payrollservice.mapDownloadPDF.has(p.id) == true" type="checkbox" name="" class="mr-3 inputcheck" (click)="onToggle(p)" checked>
```

Comment:
what is the difference between this line and the line next to it? (the 2 input field)